src/Directory.Build.targets

```diff
@@ -54,4 +54,43 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
+
+  <PropertyGroup Condition="'$(IsInvokableTestProject)' == 'true'">
+    <LaunchSettingsOutputFileFullPath Condition="'$(LaunchSettingsOutputFileFullPath)' == ''">$(AppDesignerFolder)/launchSettings.json</LaunchSettingsOutputFileFullPath>
+    <LaunchSettingsInputFileFullPath Condition="'$(LaunchSettingsInputFileFullPath)' == ''">$(ToolsDir)tests.targets</LaunchSettingsInputFileFullPath>
+
+    <CompileDependsOn>
+      GenerateLaunchSettingsFile;
+      $(CompileDependsOn);
+    </CompileDependsOn>
+  </PropertyGroup>
+
+  <!-- Create debug support files -->
+  <Target Name="GenerateLaunchSettingsFile"
+          Condition="'$(LaunchSettingsOutputFileFullPath)' != '' AND '$(OmitLaunchSettings)' != 'true'"
+          Inputs="$(LaunchSettingsInputFileFullPath)"
+          Outputs="$(LaunchSettingsOutputFileFullPath)">
+
+    <PropertyGroup>
+      <LaunchSettingsFileContent>
+{
+  &quot;profiles&quot;: {
+    &quot;xUnit.net Console&quot;: {
+      &quot;commandName&quot;: &quot;Executable&quot;,
+      &quot;executablePath&quot;: &quot;$(StartProgram.Replace('\', '\\'))&quot;,
+      &quot;commandLineArgs&quot;: &quot;$(StartArguments)&quot;,
```

Comment:
Do we need/want to do the same slash replacements on the StartArgs?

Reply:
No need to do so for xunit options and msbuild options. We only need to replace the slashes in the other two because they contain file paths.

Reply:
Ok, so long as we never pass file paths in `$(StartArguments)`.

Reply:
ok I think it makes sense to also escape it for just in case.